app/views/devise/registrations/edit.html.erb

```diff
@@ -28,16 +28,16 @@
           </div>
 
           <div class="form-actions">
-            <%= f.button :submit, "Update", class: "btn btn-primary" %>
+            <%= f.button :submit, "Update", class: "btn btn-primary float-right" %>
           </div>
         <% end %>
-        <%= link_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete, class: "btn btn-primary" %>
```

Comment:
Don't other buttons on our forms have 'outline-primary'? Just thinking we should be uniform throughout the site.

Reply:
I changed it to red since its the warning colour, i think i changed the new/edit kid forms cancel to red as well. But I’ll make them uniform later on